solutions/community-deploy/cloud/AWS/main.tf

```diff
@@ -50,7 +50,7 @@ locals {
 }
 
 resource "aws_security_group" "cortx_deploy" {
-  name        = "shailesh-multinode-poc-cortx_deploy"
+  name        = "multinode-poc-cortx_deploy"
```

Comment:
can we remove `poc` word as well from all its references in changed code?

Reply:
We should use dynamic tagging so that if two developer try this on same AWS account then terraform command should not error out

Reply:
"poc" word is removed

Reply:
dynamic tagging is fixed